libbeat/outputs/kafka/kafka.go

```diff
@@ -35,11 +38,26 @@ const (
 	defaultMaxWaitRetry = 60 * time.Second
 )
 
+var kafkaMetricsRegistryInstance metrics.Registry
+
 func init() {
 	sarama.Logger = kafkaLogger{}
+
+	reg := metrics.NewPrefixedRegistry("libbeat.kafka.")
+
+	// Note: registers /debug/metrics handler for displaying all expvar counters
+	exp.Exp(reg)
```

Comment:
Will that also include our other expvars under `/debug/metrics`? If yes, it's kind of strange to register them here, right?